aggregators/1.0/modules/ROOT/pages/aggregators-module-reference.adoc

```diff
@@ -70,10 +70,10 @@ This example aggregates elements until a pre-defined size is reached, executing
 | Name | String | The name of the aggregator. It can later be referenced by an aggregator listener. || x
 | Content | Expression | The expression that defines what to aggregate. The result of the evaluation is the value stored in the aggregation. | #[payload] |
 | Max Size | Number | The total number of elements to be aggregated before considering the aggregation complete. | | x
-| Timeout | Number |  A maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to max size, the aggregation is considered complete. A value of `0` is not supported to avoid  timing out the group constantly. | -1(UNLIMITED) |
+| Timeout | Number |  A maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to max size, the aggregation is not considered complete. A value of `0` is not supported to avoid  timing out the group constantly. | -1(UNLIMITED) |
```

Comment:
```suggestion
| Timeout | Number |  A maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to max size, the aggregation is not considered complete. A value of `0` is not supported to avoid timing out the group constantly. | -1(UNLIMITED) |
```